safe-global/web-core
# Fix: move token tracking to meta events

## What it solves

Tokens were only tracked on the Balances page although we have them on any Safe page.

I moved it to `useMetaEvents` and moved the hook itself to a separate file.

## How to test
Open any Safe and observe the `tokens` event fired immediately w/o going to the Balances page.
